python-package/setup.py

```diff
@@ -350,8 +350,8 @@ def run(self):
                   'pandas',
               ],
           },
-          maintainer='Guolin Ke',
-          maintainer_email='guolin.ke@microsoft.com',
+          #maintainer='Guolin Ke',
+          #maintainer_email='guolin.ke@microsoft.com',
```

Comment:
We need to decide what to put here. Do we already know? @saleiro

Reply:
Leaving it here in writing as per our latest meeting: maintainer should be Feedzai, and the email will be left blank as all discussion should be preferably held on GitHub instead of private emails.

Link: #27